PNA.mdk

```diff
@@ -960,14 +967,33 @@ It is expected that many PNA implementations will restrict
   names must match the hit action parameter names, and their types
   must be the same as the corresponding hit action parameters.
 
-The new entry will have the same key field values that were
-searched for in the table when the miss occurred, which caused the
-table's default action to be executed.  The action will be the one
-named by the string that is passed as the parameter `action_name`.
+The new entry will have the same key that was searched for in the table
+when the miss occurred, which caused the table's default action to be
+executed.  The action will be the one named by the string that is passed
+as the parameter `action_name`.
 
 If the attempt to add a table entry succeeds, the return value is
 `true`, otherwise `false`.
 
+An example program is included below using `add_entry`.
+
+~Begin P4Example
+action hit(bit<8> a0, bit<16> a1) {}
+action miss(bit<8> a0, bit<16> a1) {
```

Comment:
This program won't compile, I'm nearly certain, because no action parameters are provided to the action `miss` in the definition of the `default_action` below for table `foo`.

I would recommend changing the definition of `miss` so that it has 0 parameters.

You could also consider adding a link to a more complete example program such as the one in the examples directory like https://github.com/p4lang/pna/blob/main/examples/pna-example-tcp-connection-tracking.p4

Reply:
Correct. I read the P4-16 working doc and see text in `default_action` must supply arguments for the control-plane bound parameters because the `default_action` is synthesized at compilation time